views/index.hbs

```diff
@@ -0,0 +1,16 @@
+<html lang="en">
```

Comment:
you don't need to add the <head> and <body> tag when using handlebars with a layout.
Although you can add a head tag if you need a specific css stylesheet for the view.